project/chrome-extension/popup.html

```diff
@@ -1,24 +1,49 @@
 <!DOCTYPE html>
 <html>
-
-<head>
+  <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
     <title>InfoCheck</title>
     <link href="styles.css" rel="stylesheet" type="text/css" />
-    <script src="script.js"></script>
-</head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    
+  </head>
+  <body>
+    <!--Putting all of the main menu content on a div so it can be styled later.-->
+    <div id="mainMenu">
 
-<body>
     <h1>InfoCheck</h1>
 
-    <h3>Choose a topic:</h3>
+    <img src="main-menu.jpg" id="mainPicture" alt="Cartoon of a person on a computer.">
+
+    <!--List of topics the user can choose from.-->
+    <p>Choose a topic, we'll verify it for you.</p>
     <div class="infoCheck-topics">
-        <a href="example.html"><button id="covid19">COVID-19</button></a>
-        <a href="example.html"><button id="vaccines" onclick="readWord(this.id)">VACCINES</button></a>
-        <a href="example.html"><button id="elections" onclick="readWord(this.id)">ELECTIONS</button></a>
-        <a href="example.html"><button id="climateChange" onclick="readWord(this.id)">CLIMATE CHANGE</button></a>
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="rdoBtn" id="COVID-19">
+        <label class="form-check-label" for="COVID-19">
+          COVID-19
+        </label>
+        <br>
+        <input class="form-check-input" type="radio" name="rdoBtn" id="Vaccines">
+        <label class="form-check-label" for="Vaccines">
+          Vaccines
+               </label> 
+               <br>
+               <input class="form-check-input" type="radio" name="rdoBtn" id="Elections">
+        <label class="form-check-label" for="Elections">
+          Elections      
```

Comment:
We will need to add in to the text files info about these words. Just commenting